### DISCUSSION TITLE
500 Internal Server Error - while adding VC in wallet id

### DISCUSSION THREAD

**Discussion by abdurrahmaan12** [walt-id/.github#90] [Help]
When i am trying to add verifiable credential which is created through swagger ui, i am getting {"exception":true,"status":"Bad Request","code":"400","message":"Fields [credential_issuer, credential_configuration_ids] are required for type with serial name 'id.walt.oid4vc.data.CredentialOffer', but they were missing"}
I refered this documentation https://docs.walt.id/issuer/api/credential-issuance/vc-oid4vc and my deployment setup is docker 

request command:-

curl -X 'POST' \
  'http://localhost:7002/openid4vc/jwt/issue' \
  -H 'accept: text/plain' \
  -H 'Content-Type: application/json' \
  -d '{
  "issuerKey": {
    "type": "jwk",
    "jwk": {
      "kty": "EC",
      "d": "5ZBzWMk5LbDFr2oOmf1ZvVkhIJONFr6jXXP_t744fd0",
      "crv": "P-256",
      "kid": "UIGe8XEJF8mPn5XjJMibeUAjnALEQehd-PQCu8mAltM",
      "x": "g20fpVqQlA_RWyGNA9mJLilXoQksxk00deDMbO3uo0w",
      "y": "O78RSGca8zC-e2vCO_vAJDj4t70ERVSaglccCNcbVCw"
    }
  },
  "credentialConfigurationId": "UniversityDegree_jwt_vc_json",
  "credentialData": {
    "@context": [
      "https://www.w3.org/2018/credentials/v1",
      "https://www.w3.org/2018/credentials/examples/v1"
    ],
    "id": "http://example.gov/credentials/3732",
    "type": [
      "VerifiableCredential",
      "UniversityDegreeCredential"
    ],
    "issuer": {
      "id": "did:web:vc.transmute.world"
    },
    "issuanceDate": "2020-03-10T04:24:12.164Z",
    "credentialSubject": {
      "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
      "degree": {
        "type": "BachelorDegree",
        "name": "Bachelor of Science and Arts by abdur local"
      }
    }
  },
  "mapping": {
    "id": "<uuid>",
    "issuer": {
      "id": "<issuerDid>"
    },
    "credentialSubject": {
      "id": "<subjectDid>"
    },
    "issuanceDate": "<timestamp>",
    "expirationDate": "<timestamp-in:365d>"
  },
  "authenticationMethod": "PRE_AUTHORIZED",
  "issuerDid": "did:jwk:eyJrdHkiOiJFQyIsImNydiI6IlAtMjU2Iiwia2lkIjoiVUlHZThYRUpGOG1QbjVYakpNaWJlVUFqbkFMRVFlaGQtUFFDdThtQWx0TSIsIngiOiJnMjBmcFZxUWxBX1JXeUdOQTltSkxpbFhvUWtzeGswMGRlRE1iTzN1bzB3IiwieSI6Ik83OFJTR2NhOHpDLWUydkNPX3ZBSkRqNHQ3MEVSVlNhZ2xjY0NOY2JWQ3cifQ"
}'

**response:-**

openid-credential-offer://host.docker.internal:7002/?credential_offer_uri=http%3A%2F%2Fhost.docker.internal%3A7002%2Fopenid4vc%2FcredentialOffer%3Fid%3Daef93a51-be95-42e6-8071-a8a632b09bbf

**Comment by alegomes** [comment]
This forum is no longer being monitored. Please, if you still need our help, submit your question as a [Community Stack issue](https://github.com/walt-id/waltid-identity/issues).

All the best
